gradle.properties

```diff
@@ -1 +1,4 @@
 org.gradle.parallel=true
+# Avoid formatting copyrights, which are formatted in a
+# slightly non-standard way due to javapoet codegen.
+com.palantir.baseline-format.copyright=false
```

Comment:
This will opt us out of unintentionally formatting generated classes that we check in